themes/default/content/docs/intro/vs/terraform/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Pulumi vs. Terraform
+meta_desc: "Pulumi and Terraform have a few similarities, but they differ in many key ways. This page helps provide a rundown of the differences."
```

Comment:
Changed this meta description to be a bit more page-specific (using the lead sentence from the current page). Open to suggestions if folks have better ideas of course!